src/balancereader/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="console" target="SYSTEM_OUT">
+            <PatternLayout
+                pattern="%d{yyyy-MM-dd, HH:mm:ss.SSS} | [%level] | %M | %msg%n" />
```

Comment:
Might want to really quickly check this formatting against Sunny's proposed python formatter: https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/228/files#diff-fa7cce73252899ec9f378d70a9736285 

not sure if `.SSS`  is milliseconds? I think just seconds is fine.

Reply:
Example 

```
balancereader-6464775f55-fd76n balancereader 2020-06-15, 11:38:38.460 | [INFO] | log | Starting service [Tomcat]
```

Reply:
@sunny3728 can you compare this formatting to your Python log format? thanks!

Reply:
My current python format doesn't include milliseconds. There also is no comma between date and time, like there is here. Should I add both of those?